### DISCUSSION TITLE
„More by this Developer“ should show tweaks with the same email

### DISCUSSION THREAD

**Discussion by 10Meisterbaelle** [wstyres/Zebra#1979] [Ideas]
When I click on "More by this Developer", I sometimes get tweaks by different developers with the same name, so I'd suggest using the email instead of the name to show tweaks by the same developer.

**Comment by kirb** [comment]
Hmm, there might have been a reason we did it this way, but I wasn’t around when that decision was made so I wouldn’t know the reasoning. Could you possibly provide some examples so I can review what Zebra’s logic might be doing here?

**Reply by 10Meisterbaelle** [reply]
Example: Person publishes a package on a repo, sets the author to XYZ <xyz1@gmail.com>. Another person publishes a package on a different repo (or the same if the repo allows the same name for multiple people), sets the author to XYZ <xyz2@gmail.com>. Zebra will consider these as the same author.

**Reply by kirb** [reply]
Sorry, to clarify I meant is there a specific example you have where you see this? I want to review what’s going on with those exact packages you noticed are affected.
